Airdrop.sol

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import "./Token.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+contract Airdrop is Ownable{
+
+    Token public token;
+
+    constructor(address payable _token) {
+        token = Token(_token);
+    }
+
+    function depositEther() external payable onlyOwner {
+
+    }
+
+    function dropEther(
+        address payable[] calldata _recipients,
+        uint256 _amount
+    ) external payable onlyOwner {

```

Comment:
are you sure this function has recive  eth?

Reply:
remove payable, because you send ethe in depositEther function;
this modifier is not nessesary